pkg/operator/ceph/cluster/osd/provision_spec.go

```diff
@@ -67,6 +67,7 @@ func (c *Cluster) makeJob(osdProps osdProperties, provisionConfig *provisionConf
 
 	if osdProps.onPVC() {
 		k8sutil.AddLabelToJob(OSDOverPVCLabelKey, osdProps.pvc.ClaimName, job)
+		k8sutil.AddLabelToPod(OSDDeviceClassSetLabelKey, osdProps.name, &job.Spec.Template)
```

Comment:
Let's add it to both the job and the pod spec.

Reply:
Done.